api/server/apps_test.go

```diff
@@ -288,7 +288,7 @@ func TestAppUpdate(t *testing.T) {
 		expectedError error
 	}{
 		// errors
-		{datastore.NewMock(), logs.NewMock(), "/v1/apps/myapp", ``, http.StatusNotFound, models.ErrInvalidJSON},
+		{datastore.NewMock(), logs.NewMock(), "/v1/apps/myapp", ``, http.StatusNotFound, models.ErrAppsNotFound},
```

Comment:
this app exists in the mock datastore, this should work (perhaps it needs an id?)

Reply:
it uses empty datastore, so the app is not there, will use `ds` from few lines above, then the test will remain as it was.